api/translation.go

```diff
@@ -43,3 +43,32 @@ func ProfileToProto(in *PublicProfile) *pb.Profile {
 		Description: in.Description,
 	}
 }
+
+func UserActivityReportToProto(in *UserActivityReport) *pb.GetUserActivityReportResponse {
+	response := &pb.GetUserActivityReportResponse{
+		Channels: make(map[string]*pb.ChannelData),
+	}
+
+	for channelID, activity := range *in {
+		channelData := &pb.ChannelData{
+			TotalMessages: int32(activity.TotalMessages),
+		}
+		if activity.MessageCountDistribution != nil {
+			channelData.MessageCountDistribution = messageCountDistributionToProto(activity.MessageCountDistribution)
+		}
+		response.Channels[channelID] = channelData
```

Comment:
```suggestion
		response.Channels[channelID] = &pb.ChannelData{
			TotalMessages:            int32(activity.TotalMessages),
			MessageCountDistribution: messageCountDistributionToProto(activity.MessageCountDistribution),
		}
```
It can be simplified, because `messageCountDistributionToProto` handles nil slices

Reply:
done in [apply review suggestions](https://github.com/kacperf531/sockchat/pull/11/commits/4054b76b6854361e07ae3021db901a9e9bd32cb8)